README.md

```diff
@@ -471,15 +471,15 @@ unsafe fn XCAR(object: LispObject) -> LispObject {
 // of XCAR: it only passes valid cons cells. We just use
 // unsafe blocks instead.
 fn car(object: LispObject) -> LispObject {
```

Comment:
We no longer need `unsafe`. Time for a new example!

    if let Some(cons) = object.as_cons() {
        cons.car()
    } else if object.is_nil() {
        LispObject::constant_nil()
    } else {
        wrong_type!(Qlistp, object)
    }

Reply:
There are not many "clean" examples of unsafe. We have wrapped clean APIs on a lot of it. Maybe an example showing access to `globals`? Or an example using the accessor functions to get data out of structs?

Reply:
multibyte.rs has a fair amount of slice creation and what not.

Reply:
OK, I've updated the example to show dealing with strings, based on our wrapped safe APIs. I wanted to focus on pointers as a fundamental source of unsafety.